content/docs/getting-started/db.md

```diff
@@ -176,6 +176,19 @@ flexibly define the future internally.
 
 [`BoxFuture`]: https://docs.rs/futures/0.1/futures/future/type.BoxFuture.html
 
+
+Well also need a `Message` struct that we can use to serialize the row that we
+will be getting from the database into json using `serde`:
```

Comment:
I'd be fine either way, but if you'd like to send a follow-up PR please feel free!